### PR TITLE
Feature/mv unused dev

### PR DESCRIPTION
**Pull request by XianwuXue-NOAA**
Summarize of the changes:

1) Move unused module files, parm/gefs_awips, scripts, ush, util and sorc files to rocoto/bun/gefs_dev
2) Update and cleanup the pyGEFS to only keep the used tasks/programs
3) Move j-job, ex-script, gempak/fix[ush] scripts of task avg_gempak_vgf to rocoto/bun/gefs_dev
4) Remove the 'goto' in sorc/gefs_ensstat.fd/EPDF.f
5) Remove 'COMINgfs' in jobs/JGEFS_ATMOS_GEMPAK_META
6) Add more variables to rocoto/bin/wcoss_dell_p35/common.sh
7) Improve j-job and ush script of postsnd task
8) Improve ush script of prdgen task

### CODE REVIEW

File: sorc/gefs_ensstat.fd/EPDF.f [hunk 1]
```diff
@@ -240,10 +240,10 @@ function epdf(ef,ew,n,qq,ictl)
       do i = 1, m
        if (qq.lt.q(i)) then
         k=i-1
-        goto 101
+        exit
```
Comment (WalterKolczynski-NOAA) [inline sorc/gefs_ensstat.fd/EPDF.f:243]: This is functionally different than the control. Are we sure this performs as desired?

**Review by WalterKolczynski-NOAA — COMMENTED** [review]
See comments

File: sorc/gefs_ensstat.fd/EPDF.f (hunk 1 above)
Comment (WalterKolczynski-NOAA) [reply sorc/gefs_ensstat.fd/EPDF.f:243]: Nevermind. Not used to dealing with FORTRAN loops.

File: ush/gefs_sndp.sh [hunk 2]
```diff
@@ -61,7 +61,7 @@ EOF
 
     if [ $SENDCOM = 'YES' ]
     then 
-      if [ $SENDDBN = 'YES' ] ; then
+      if [ $SENDDBN_NTC = 'YES' ] ; then
```
Comment (WalterKolczynski-NOAA) [inline ush/gefs_sndp.sh:64]: Why the change? It seems counter to the request to change from SENDDBN_GB2. Is it confirmed that NCO approves of this change?

**Review by WalterKolczynski-NOAA — COMMENTED** [review]
One more comment

File: ush/gefs_sndp.sh (hunk 2 above)
Comment (XianwuXue-NOAA) [reply ush/gefs_sndp.sh:64]: Add Dingchen to the reviewer, This is what he changed.

Comment (WalterKolczynski-NOAA) [reply ush/gefs_sndp.sh:64]: Rereading the Implementation Standards, this seems correct. But I am going to double-check with Steven in the morning.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Review by XianwuXue-NOAA — COMMENTED** [review]
Look like, I can not approve it. So I used the comment to approve it.
